pkg/v1/providers/infrastructure-vsphere/ytt/vsphere-overlay.yaml

```diff
@@ -1 +1,228 @@
-#! Please add any overlays specific to vSphere provider under this file.
\ No newline at end of file
+#! Please add any overlays specific to vSphere provider under this file.
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+
+#@ load("lib/helpers.star", "get_bom_data_for_tkr_name", "get_default_tkg_bom_data", "kubeadm_image_repo", "get_image_repo_for_component", "get_vsphere_thumbprint")
+
+#@ load("lib/validate.star", "validate_configuration")
+#@ load("@ytt:yaml", "yaml")
+#@ validate_configuration("vsphere")
+
+#@ bomDataForK8sVersion = get_bom_data_for_tkr_name()
+
+#@ if data.values.CLUSTER_PLAN == "prod":
```

Comment:
Question: Will `prod` plan have cluster deployed across 3 AZs by default ?

Reply:
No..Prod will have 3 machine deployments by default. But if you specify the AZs, then they can be deployed across the AZs specified.